Exercises_lvl_1.py

```diff
@@ -0,0 +1,23 @@
+import random
```

Comment:
Jeśli ten skrpyt wykonuje jakieś konkretne zadanie , w tym przypadku (Skrypt wykonujący 'Guessnumber') nazwa pliku powinna to odwzierciedlać. Usunąć komentarze, zmienić nazwe pliku